### ISSUE TITLE
Exposure control

### ISSUE THREAD

**Issue by martimorta-at-work** [i3drobotics/deimos-ros#10]
Hi,
it would be great if the auto exposure and exposure control (through dynamic reconfigure ideally) was available.
Thanks!

**Comment by martimorta-at-work** [comment]
From the UVC camera, there are the options, my suggestion is adding these in dynamic reconfigure as an enum.

uvc_cam.h:65
```cpp
	V4L2_UVC_EXPOSURE_MANUAL = 1,
	V4L2_UVC_EXPOSURE_AUTO = 2,
	V4L2_UVC_EXPOSURE_SHUTTER_PRIORITY = 4,
	V4L2_UVC_EXPOSURE_APERTURE_PRIORITY = 8
```

**Comment by jveitchmichaelis** [comment]
Yep this is sensible. Although there is no priority selection, the camera is fixed aperture, so it's auto on/off and exposure time.

**Comment by martimorta-at-work** [comment]
It would be more to understand what exposure means, If @bknight-i3drobotics takes this into account for the new generic driver that would be enough, just to take it into account! Cheers
